internal/widget/base.go

```diff
@@ -53,6 +53,9 @@ func (w *Base) Position() fyne.Position {
 // Move the widget to a new position, relative to its parent.
 // Note this should not be used if the widget is being managed by a Layout within a Container.
 func (w *Base) Move(pos fyne.Position) {
+	if w.Position() == pos {
+		return
+	}
 	w.position = pos
 
 	Repaint(w.super())
```

Comment:
This is a bit nitpicky but would you mind updating the two base widgets to have the lines match in terms of empty lines? I.e. move the line setting the position down one line and keep the empty line between the if statement instead.